code/lib/core-events/package.json

```diff
@@ -81,6 +81,7 @@
     "ts-dedent": "^2.0.0"
   },
   "devDependencies": {
+    "chalk": "^4.1.0",
```

Comment:
could this bring potential problems for the core-events package? as chalk would be prebundled in something that goes to the browser as well

Reply:
How, could it get pre-bundled into something that goes into the browser, due to this change?

Reply:
AFAIK this doesn't have an affect on any code we ship to the browser, as long as that browser code doesn't reference `chalk`.

I had to move chalk into here as @vanessayuenn asked me to move this to `core-events`: https://github.com/storybookjs/storybook/pull/26035#discussion_r1489848271

Reply:
Also have a funny feeling about adding `chalk` to that package, but I'll leave that decision to you